src/CalculatorFoodItem1/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import classNames from 'classnames';
+import RadioButton from '../RadioButton';
+import Spacer from '../Spacer';
+import Typography from '../Typography';
+import Wrapper from '../Wrapper';
+import styles from './styles.css';
+
+function CalculatorFoodItem1(props) {
+  return (
+    <div>
+      <Wrapper
+        className={styles.wrapper}
+        size="standard"
+        >
+        <RadioButton
+          checked={props.checkedEatAmount.toLowerCase() === 'little'}
+          className={classNames([[props.className]: props.className])}
```

Comment:
Needs to pass in an object and not array to classNames. Should be: className={classNames(**{** [props.className]: props.className**}**)}

Needs to be fixed all places :-)